git-cliff-core/src/changelog.rs

```diff
@@ -612,6 +605,33 @@ impl<'a> Changelog<'a> {
 	}
 }
 
+fn get_body_template(config: &Config) -> String {
+	let body_template = config
+		.changelog
+		.body
+		.as_deref()
+		.unwrap_or_default()
+		.to_string();
+	warn_on_deprecated_commit_vars(&body_template, &[
+		"github",
+		"gitea",
+		"gitlab",
+		"bitbucket",
+	]);
+	body_template
+}
+
+fn warn_on_deprecated_commit_vars(body_template: &str, vars: &[&str]) {
+	for var in vars {
+		if body_template.contains(&format!("commit.{var}")) {
```

Comment:
Unfortunately performing a string check here is not going to be reliable because it is going to print out a warning even though I just use a "commit.github" as a string in the template.

Instead, we should use the `contains_variable` function which does a proper check in the template variables :)

Reply:
done in [5a67a20](https://github.com/orhun/git-cliff/pull/822/commits/5a67a2066761ea40edb05bc7634a27a4e5841fef)